### PR TITLE
ACA-1602: Removing SKU tier, capacity and size parameters from role and test

### PR DESCRIPTION
**Pull request by Yaish25491**
Removing SKU tier, capacity, and size parameters from role and test

### CODE REVIEW

File: roles/azure_manage_postgresql/tasks/create.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     source_server_id: "{{ azure_manage_postgresql_postgresql_source_server_id | default(omit) }}"
     restore_point_in_time: "{{ azure_manage_postgresql_manage_postgresql_postgresql_restore_point_in_time | default(omit) }}"
     tags: "{{ azure_manage_postgresql_tags | default(omit) }}"
-
+
```
Comment (nirarg) [inline roles/azure_manage_postgresql/tasks/create.yml:58]: Please remove this

File: changelogs/fragments/20240721-azure_manage_postgresql.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - Removing tier, capacity, and size from SKU parameters from README file, both from Role Variables and Example Playbook
+  - Removing azure_manage_postgresql_postgresql_sku.tier from defalts
+  - Removing sku.tier from assert testing
```
Comment (nirarg) [inline changelogs/fragments/20240721-azure_manage_postgresql.yml:4]: I don't think these description needed in the release notes, this is implicit from the first one

**Review by nirarg — APPROVED** [review]

File: changelogs/fragments/20240721-azure_manage_postgresql.yml (hunk 2 above)
Comment (Yaish25491) [reply changelogs/fragments/20240721-azure_manage_postgresql.yml]: removed

File: roles/azure_manage_postgresql/tasks/create.yml (hunk 1 above)
Comment (Yaish25491) [reply roles/azure_manage_postgresql/tasks/create.yml]: fixed

**Review by p3ck — APPROVED** [review]

**Pull request merged**